test/test.js

```diff
@@ -0,0 +1,35 @@
+const test = require('ava');
+const cfntest = require('@cfn-modules/test');
+
+test('defaults', async t => {
```

Comment:
`test(` => `test.serial(`

Reply:
@michaelwittig Could you please explain why it is important to run the test in serial?

Reply:
ava, the test libraray, cannot limit the number of tests that run in parallel if they are defined in a single file. ava can only limit the number of test files that execute in parallel (set to 4). To avoid AWS rate limits, I usually define the tests in a file as serial but you can split it up into multiple files for concurrency.